src/main/java/net/sf/jabref/MetaData.java

```diff
@@ -158,18 +158,23 @@ public void putData(String key, Vector<String> orderedData) {
     /**
      * Look up the directory set up for the given field type for this database.
      * If no directory is set up, return that defined in global preferences.
+     * There can be up to three directory definitions for these files:
+     * the database's metadata can specify a general directory and/or a user-specific directory
+     * or the preferences can specify one.
+     *
+     * The settings are prioritized in the following order and the first defined setting is used:
+     * 1. metadata user-specific directory
```

Comment:
What is the difference between metadata user-specific and general? I get the one with the preferences directory.